src/js/components/action.jsx

```diff
@@ -131,15 +131,16 @@ export class ActionComponent extends Component {
                     );
             }
         } else if (this.props.type === 'postback') {
-            const text = this.state.state === 'processing' ?
+            const isProcessing = this.state.state === 'processing';
+            const text = isProcessing ?
                 <LoadingComponent /> :
                 this.props.text;
 
             return (
                 <div className='sk-action'>
                     <button className='btn btn-sk-primary'
                             style={ style }
-                            onClick={ this.onPostbackClick }>
+                            onClick={ !isProcessing && this.onPostbackClick }>
```

Comment:
React doesn't require this to be a function? Do you get warnings when onClick is `undefined`?